tests/model/liveselection.js

```diff
@@ -872,4 +872,37 @@ describe( 'LiveSelection', () => {
 			expect( values ).to.deep.equal( [] );
 		} );
 	} );
+
+	describe( '_updateAttributes()', () => {
```

Comment:
Always try think not only about the case you want to fix but also about the cases you don't want to break.